Pages/JoinInPage.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Mars_Luiz.Utilities;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mars_Luiz.Pages
+{
+    internal class JoinInPage
+    {
+        public void JoinIN(IWebDriver driver)
+        {
```

Comment:
Please implement the POM Pattern, please see the below 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
please change this and apply this to all pages